integrations/launchdarkly/.port/resources/blueprints.json

```diff
@@ -153,5 +153,32 @@
         "many": false
       }
     }
+  },
+  {
+    "identifier": "featureFlagInEnvironment",
+    "description": "This blueprint represents a feature flag in LaunchDarkly Environment.",
+    "title": "Feature Flag In Environment",
+    "icon": "Launchdarkly",
+    "schema": {
+      "properties": {},
```

Comment:
This is missing a property for the actual "state" of the flag (active, inactive, current value, whatever is appropriate and reported from the LD API)

Reply:
This will require making a call to  launchdarkly's [List feature flag statuses](https://apidocs.launchdarkly.com/tag/Feature-flags#operation/getFeatureFlagStatuses) endpoint.